backend/app/adapter/sqldb/user_short_link_integration_test.go

```diff
@@ -114,6 +114,128 @@ func TestListShortLinkSql_FindAliasesByUser(t *testing.T) {
 	}
 }
 
+func TestListShortLinkSql_IsAliasOwnedByUser(t *testing.T) {
+	now := mustParseTime(t, "2019-05-01T08:02:16Z")
+	user := entity.User{
+		ID:             "test",
+		Name:           "mockedUser",
+		Email:          "test@example.com",
+		LastSignedInAt: &now,
+		CreatedAt:      &now,
+		UpdatedAt:      &now,
+	}
+
+	testCases := []struct {
+		name               string
+		userTableRows      []userTableRow
+		shortLinkTableRows []shortLinkTableRow
+		relationTableRows  []userShortLinkTableRow
+		alias              string
+		user               entity.User
+		isFound            bool
+	}{
+		{
+			name: "alias does not exist",
```

Comment:
Let's add a case when user does not exist.

Reply:
I'm not sure what you mean by this. The relation table cannot have an entry without an exisiting user because of the foreign key usage. Do you mean when the user entity is not set in the test case?

Reply:
@oatovar Awesome! Please ignore this comment then.